charts/cluster/templates/pooler.yaml

```diff
@@ -1,21 +1,25 @@
-{{ if .Values.pooler.enabled }}
+{{ if gt (len .Values.poolers) 0 }}
+{{ $context := . -}}
```

Comment:
No need to cache the context. Inside the `range` you can access the top context by using `$`. Example: `$.Values`

Reply:
To note: I taken it from scheduled backups, I think we should change it there as well?